docs/Using-Data-Access-Tokens.md

```diff
@@ -0,0 +1,10 @@
+#### What are Data Access Tokens?
+The cBioPortal provides access to data within its database through the cBioPortal Web API.  While most users interact with this web service indirectly while visiting cBioPortal web pages, it is possible to access this web service directly.  Direct access to the web service is appealing for those users with an interest in mining or analyzing cBioPortal data beyond the confines of a web browser, such as from ‘R’ or Python clients.
+
+To access the cBioPortal web services (on installations which require login), clients will need to obtain a data access token and present this token with each web service request.
+
+#### Making Requests Using Data Access Tokens
```

Comment:
I would remove this header in the context of the existing Web API page.

Reply:
Done